CHANGELOG

```diff
@@ -9,7 +9,8 @@ v 7.13.0 (unreleased)
   - Remove link leading to a 404 error in Deploy Keys page (Stan Hu)
   - Add support for unlocking users in admin settings (Stan Hu)
   - Add Irker service configuration options (Stan Hu)
-  - Fix order of issues imported from GitHub (Hiroyuki Sato)
+  - Add "replace" and "upload" functionalities to allow user replace existing file and upload new file into current repository
```

Comment:
@stanhu thanks, I am writing the test of file upload with dropzone, but without success yet. Do you have any suggestions?